extensions/sql-format/package.json

```
@@ -0,0 +1,95 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sql-format",
+  "title": "Format SQL",
+  "description": "Format a SQL cluase use stanard SQL format",
+  "icon": "extension-icon.png",
+  "author": "tonngw",
+  "categories": [
+    "Data",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Format Clipboard SQL",
+      "description": "Formats a SQL cluase string stored in clipboard and copies it back",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "default": false,
+      "description": "Paste the formatted JSON to an active app",
+      "label": "Automatically paste the formatted JSON",
+      "name": "autopaste",
+      "required": false,
+      "title": "Auto Paste",
+      "type": "checkbox"
+    },
+    {
+      "data": [
+        {
+          "title": "Default: 2",
+          "value": "2"
+        },
+        {
+          "title": "Spaces: 4",
+          "value": "4"
+        },
+        {
+          "title": "Tab: 8",
+          "value": "tab"
+        }
+      ],
+      "description": "Output sql tabWidth",
+      "name": "tabWidth",
+      "required": false,
+      "title": "tabWidth",
```

Title(s) should generally follow title case and defaults can be provided for preference, rather than specifying in name. Raycast will mark the default in this case.

```suggestion
      "title": "Tab Width",
      "default": "2",
```